.travis.yml

```diff
@@ -78,7 +78,7 @@ deploy:
   verbose: true
   on:
     branch: master
-    condition: $GDALVERSION="3.0.2" && $TRAVIS_PYTHON_VERSION=3.8.*
```

Comment:
Does the deleted line not ensure that only one of the 4 travis runs builds the deployment? Don't we need that?

Reply:
There is only one python 3.8 Travis env meaning only one build will push the docs in master.